src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -1518,6 +1518,79 @@ GenTree* Lowering::LowerHWIntrinsic(GenTreeHWIntrinsic* node)
             break;
         case NI_Sve_ConditionalSelect:
             return LowerHWIntrinsicCndSel(node);
+        case NI_Sve_SetFfr:
+        {
+            StoreFFRValue(node);
+            break;
+        }
+        case NI_Sve_GetFfrByte:
+        case NI_Sve_GetFfrInt16:
+        case NI_Sve_GetFfrInt32:
+        case NI_Sve_GetFfrInt64:
+        case NI_Sve_GetFfrSByte:
+        case NI_Sve_GetFfrUInt16:
+        case NI_Sve_GetFfrUInt32:
+        case NI_Sve_GetFfrUInt64:
+        {
+            LIR::Use use;
+            bool     foundUse = BlockRange().TryGetUse(node, &use);
+            if (foundUse)
+            {
+                // Remove GetFfr and replace the GetFfr's use with the lvaFfrRegister lclVar.
+                CreateFFRDefIfNeeded(node);
+
+                GenTree* lclVar = comp->gtNewLclvNode(comp->lvaFfrRegister, TYP_MASK);
+                BlockRange().InsertBefore(node, lclVar);
+                use.ReplaceWith(lclVar);
+                GenTree* next = node->gtNext;
+                BlockRange().Remove(node);
+                return next;
+            }
+            else
+            {
+                node->SetUnusedValue();
+            }
+
+            break;
+        }
+        case NI_Sve_LoadVectorFirstFaulting:
+        {
+            unsigned lclNum = comp->lvaFfrRegister;
+
+            LIR::Use use;
+            bool     foundUse = BlockRange().TryGetUse(node, &use);
+
+            if (foundUse)
```

Comment:
I think we have to create FFR uses/defs even if `!foundUse`. The use is just for the value, but the user could potentially use `LoadVectorFirstFaulting` as a way to probe inaccessible memory as well, without storing the value anywhere.

Reply:
> as a way to probe inaccessible memory as well

That's a fair point.